TodoApp.ConsoleApp/Services/TodoService.cs

```diff
@@ -1,77 +1,87 @@
 ﻿using System.Collections.Generic;
-using TodoApp.Library.Data;
-using TodoApp.Library.Models;
+using TodoApp.ConsoleApp.Repositories.Models;
+using TodoApp.ConsoleApp.Repositories.XMLRepository;
 using System.Linq;
 using System;
+using System.IO;
+using System.Xml;
+using System.Text;
 
 namespace TodoApp.ConsoleApp.Services
 {
     public class TodoService : ITodoService
```

Comment:
To every method that modifies the data (create, update, delete) add `Repository.Save()` so we actually save the data modification.